### PR TITLE
this is the neweval that I did

### PR DESCRIPTION
**Pull request by Berek135**

### CODE REVIEW

**Comment by Tesfit12** [comment]
hi bereke it looks nice, 👍

**Comment by Berek135** [comment]
Good thanks I appreciate it!

File: cal.html [hunk 1]
```diff
@@ -22,31 +21,27 @@
...
              <td><input type="button" class="operator" name="div" value="/" onclick="calculator.display.value += '/'"></td>
-            
-             
        </tr>     
     </table>
  </form>
  </body>
- </html>
\ No newline at end of file
+ </html>
```
Comment (Tesfit12) [inline cal.html]: you do not need twice html

File: cal.html [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-         
-                <title>MyCalculator</title>
-                
+   <head>
+      <title>MyCalculator</title>
                 <link rel="stylesheet" type ="text/css" href ="style.css"></link>
                 <script type="text/javascript" src="javas.js"></script>
+                <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjs/6.2.2/math.js"></script>
   
-</head>
```
Comment (Tesfit12) [inline cal.html]: on this line the indentation is not good.

**Review by Tesfit12 — APPROVED** [review]

**Pull request merged**